week3/src/components/main/FirstStage.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./index.css";
+const FirstStage = ({ setRecommendationType, setFirstChoice, firstChoice }) => {
+  return (
+    <>
+      <button
+        onClick={() => {
+          setFirstChoice("korea");
+        }}
+        className={firstChoice === "korea" ? "selected" : ""}
```

Comment:
이거 이항선택자로 해도 될 것 같아요! 
```suggestion
        className={firstChoice === "korea" && "selected" }
```